File: jaegr-persistence/src/main/java/com/jaegr/model/UserView.java
package com.jaegr.model;

import com.jaegr.DBUser;

/**
 * Created by jonas on 30.06.17.
 */
public class UserView {
    private String name;
    private long id;
    private boolean disabled;
    private boolean isAdmin;

    public UserView(DBUser user) {
        this.name = user.getName();
        this.id = user.getId();
        this.disabled = user.isDisabled();
        this.isAdmin = user.isAdmin();
    }

    public String getName() {
        return name;
    }

    public long getId() {
        return id;
    }

    public boolean isDisabled() {
        return disabled;
    }

    public boolean isAdmin() {
        return isAdmin;
    }
}

File: jaegr-presentation/src/main/dart/lib/model/UserView.dart

class User {
  String name;
  int id;
  bool disabled;
  bool isAdmin;
  User (this.name, this.id, this.disabled, this.isAdmin);
  User.fromJson(Map json){
    this.id = json["id"];
    this.disabled = json["disabled"];
    this.name = json["name"];
    this.isAdmin = json["isAdmin"];
  }

}